jlm/llvm/opt/DeadNodeElimination.cpp

```diff
@@ -470,40 +473,36 @@ DeadNodeElimination::SweepLambda(rvsdg::LambdaNode & lambdaNode) const
 }
 
 void
-DeadNodeElimination::SweepPhi(phi::node & phiNode) const
+DeadNodeElimination::SweepPhi(rvsdg::PhiNode & phiNode) const
 {
-  util::HashSet<const rvsdg::RegionArgument *> deadRecursionArguments;
+  util::HashSet<const rvsdg::output *> deadRecursionArguments;
```

Comment:
This definition is dead, no? I do not see it used anywhere.

Reply:
indeed